### ISSUE TITLE
Feature request: convert state from HTML

### ISSUE THREAD

**Issue by ZeroCho** [jpuri/react-draft-wysiwyg#103]
Thanks for great editor!
Currently it works well when writing a new post, but when I try to edit posts, some tags are not converted to `editorState` properly.
For example, `iframe` tag isn't converted to editorState using `draft-js` 's basic `convertFromHTML` method.
I think this editor will be better if it supports converting tags, which are used by this editor, into `editorState`.

**Comment by jpuri** [comment]
@ZeroCho: I see, I will work on it. But I can not promise how soon as this is comparatively bigger requirement. But you will definitely hear back from me.

**Comment by jpuri** [comment]
@ZeroCho: why do not you consider saving editor state as JSON ?

**Comment by ZeroCho** [comment]
@jpuri Because i already wrote 100+ posts in my blog. All of them are saved in HTML. So I cannot take risk to convert all of them into JSON.

**Comment by andreslucas** [comment]
I include a way in [https://andresin87.github.io/wysiwyg/](https://andresin87.github.io/wysiwyg/) under [import HTML] story

**Comment by ZeroCho** [comment]
@andreslucas Well, I'm afraid that your import HTML doesn't work
I typed this below code, and it doesn't render correctly.
````
<p>test</p>
<iframe src="//codepen.io/zerocho/embed/mOMyJb/?height=265&theme-id=0&default-tab=html,result&embed-version=2" height="315px" width="100%" />
<p>Yeah</p>
```

**Comment by andreslucas** [comment]
You're right. I'll review what is happening. I'm using [https://github.com/sstur/draft-js-import-html](https://github.com/sstur/draft-js-import-html) as a dependency.

**Comment by shikelong** [comment]
draft-js-import-html lost inline style when convert html to contentState。

**Comment by jpuri** [comment]
I am realizing this is one most wanted thing to have lib to convert html to editorState, I am prioritizing it and starting on coding for it.

**Comment by vhmth** [comment]
Thank you @jpuri :-)

**Comment by jpuri** [comment]
I just published this on npm `html-to-draftjs@0.0.1-beta`. Plz not atm it can take care of only html generated by react-draft-wysiwyg.

I am open to extend it for more common use cases.
